src/translations/en.ts

```diff
@@ -1509,6 +1509,11 @@ export default {
 		"yourCalendars_label": "Your calendars",
 		"yourFolders_action": "YOUR FOLDERS",
 		"yourMessage_label": "Your message",
-		"you_label": "You"
+		"you_label": "You",
+		"invalidDate_msg": "Invalid Date",
+		"pre1970Start_msg": "Dates earlier than 1970 are currently outside the valid range",
```

Comment:
"currently" is a bit redundant.

I think that the separate keys for import are also a bit redundant but I don't have a strong opinion.